nixos/modules/tasks/trackpoint.nix

```diff
@@ -55,6 +55,15 @@ with lib;
         '';
       };
 
+      device = mkOption {
+        default = "TPPS/2 IBM TrackPoint";
+        type = types.str;
+        description = ''
+          The device name of the trackpoint. You can check with xinput.
+          Some newer devices (example x1c6) use "TPPS/2 Elan TrackPoint.";
```

Comment:
Umm, I meant like `...(example x1c6) use "TPPS/2 Elan TrackPoint".` :)

Reply:
:man_facepalming:

 I fixed it now, I also removed the `;`